### PR TITLE
feat: add onReady property

### PR DESCRIPTION
**Pull request by vault-developer**
Hello.
Thank you for this library - it helps me a lot!
I guess it would be great feature to add "onReady" property - to have possibility to use editor modifiactions callback.
https://editorjs.io/configuration#editor-modifications-callback
It can be useful with other editorjs plugins. (for example this one - https://github.com/kommitters/editorjs-undo)

### CODE REVIEW

File: lib/EditorJs.tsx [hunk 1]
```diff
@@ -62,13 +63,21 @@ class EditorJsContainer extends React.PureComponent<Props> {
     onChange(api, newData)
   }
 
+  handleReady = () => {
+    const { onReady } = this.props
+    if (!onReady) return
```
Comment (dipiash) [inline lib/EditorJs.tsx:68]: This repo use code style with brackets for `if (...) { ... }` conditions

**Review by dipiash — CHANGES_REQUESTED** [review]

File: lib/EditorJs.tsx (hunk 1 above)
Comment (vault-developer) [reply lib/EditorJs.tsx]: Thank you, you are right. I've updated this line.

**Review by dipiash — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 node_modules
 .rpt2_cache
+.idea
```
Comment (Jungwoo-An) [inline .gitignore:3]: Please delete this line. `.idea` directory is required only InteliJ users.

**Review by Jungwoo-An — COMMENTED** [review]
Wow! First-time contributor! 🤝

This proposal is very awesome. If you don't mind, can you update [README.md](https://github.com/Jungwoo-An/react-editor-js#-api)?

File: .gitignore (hunk 2 above)
Comment (vault-developer) [reply .gitignore]: Sure, I deleted `.idea` and updated info in [README.md](https://github.com/Jungwoo-An/react-editor-js#-api).
Thank you for reviewing this PR.

File: README.md [hunk 3]
```diff
@@ -60,6 +60,7 @@ Allow all options of [editor-js](https://github.com/codex-team/editor.js/blob/ma
 | enableReInitialize | Boolean                                                                         | Change editor-js data when componentDidUpdate                      |
 | onChange           | (api: API, newData: OutputData) => void                                         | Fires when something changed in DOM                                |
 | onCompareBlocks    | (newBlocks?: OutputData['blocks'], oldBlocks?: OutputData['blocks']) => boolean | Use to avoid Infinite update when enableReInitialize used with onChange ([Recommended Library](https://github.com/FormidableLabs/react-fast-compare)) |
+| onReady            | (instance?: EditorJS) => void                                                   | Use to execute callback when editor-js instance has initialized    |
```
Comment (Jungwoo-An) [inline README.md:63]: 👍

**Review by Jungwoo-An — APPROVED** [review]
I'll merge this pr. Thank you for your nice work 👍

**Pull request merged**